src/SeleniumLibrary/locators/windowmanager.py

```diff
@@ -39,7 +39,45 @@ def __init__(self, ctx):
             'default': self._select_by_default
         }
 
-    def get_window_infos(self):
+    def get_window_handles(self, browser):
+        if is_string(browser) and browser == 'ALL':
+            handles = []
+            current_index = self.drivers.current_index
+            for index, driver in enumerate(self.drivers, 1):
```

Comment:
Hmm, are you sure that this will work, because drivers is a [set](https://docs.python.org/3.7/library/stdtypes.html#set-types-set-frozenset)

Reply:
But set is iterable..

class set([iterable])

Or what exactly are you afraid is not working?

Reply:
My bad, drivers are not in set. Please ignore